docs/book/drafts/organization.md

```diff
@@ -65,7 +65,7 @@ How this guide is organized:
 - [Contributing Guide](../guidelines/contributing.md)
 
 ZenBytes -> A mlops course using ZenML?
-ZenFiles -> Production-ready pipelines (ZenTemplates?)
+ZenML Projects -> Production-ready pipelines (ZenTemplates?)
```

Comment:
I think this entire file can / should be deleted. Confusing and it doesn't really add anything. (Even if it's in drafts).